test/cli/simple-globals.json

```diff
@@ -0,0 +1,11 @@
+[{
```

Comment:
Can we talk to @abhijitkane and see if these can be unified and if there's a reason why they are not same format.